CONTRIBUTING.md

```diff
@@ -37,193 +37,210 @@ To add a new page to the website, [create a new markdown file with YAML frontmat
 
 ```yaml
 ---
+# Name of the product (mandatory).
 title: Timeturner
-# Possible values are os,db,app,lang,framework,device,service,server-app
+
+# Category of the product (mandatory).
+# Possible values are os,db,app,lang,framework,device,service,server-app.
 # If you add a new value, please mention it on the PR Description. Some rough guidelines:
-# os is operating systems (and similar projects)
-# db is databases, os is operating systems, lang is programming languages
-# app is end-user applications
-# lang is programming languages
-# framework is used for application libraries, SDKs, frameworks etc
-# device is physical devices
-# service is managed service offerings (SaaS/PaaS etc)
-# server-app are applications usually installed on the server-side
+# - os is operating systems (and similar projects)
+# - db is databases, os is operating systems, lang is programming languages
```

Comment:
isnt this also contains explanation of line above and below ?

Reply:
You are right, I did not pay enough attention.

Reply:
Fixed.